extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -146,6 +154,20 @@ describe.skip('ProjectsController: project controller operations', function ():
 			await dialog.generateScriptClick();
 
 			should(holler).equal(generateHoller, 'executionCallback() is supposed to have been setup and called for GenerateScript scenario');
+
+			await projController.object.readPublishProfile(vscode.Uri.parse('test'));
```

Comment:
Should the dialog be reset beforehand like the `deployClick()` and `generateScriptClick()` tests?

Reply:
Are the deployClick() and generateScriptClick() tests resetting the dialog just to make sure nothing got changed?

Reply:
Yeah - to make sure that a clean state is getting tested instead of a potential dirty state from earlier side-effects.  Makes it more unit-testy.